Apple Home page clone

Nkuria opened the pull request:
This is a clone for the previous apple home page created using HTML and CSS

m-anwar97 committed:
added html and css file

Nkuria committed:
Worked on the header part

Nkuria committed:
Corrected path errors

m-anwar97 committed:
added main content and footer/ added navbar

Nkuria committed:
Worked on read me file and part of navigation bar

Nkuria committed:
Fixed css linter issues

m-anwar97 committed:
attempt to fix css errors

m-anwar97 committed:
final css error fix

SpaYco commented on README.md:
```diff
@@ -1,2 +1,30 @@
+👤 **Mina Ibrahim**
+
+- Github: [@githubhandle](https://github.com/coldatlas)
+
```
- [x] [Required] your README can be more descriptive ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
 also change the '@githubhandle' to your actual handle for example mine is @Spayco, you can also add the rest of your social media profiles, but that's optional

SpaYco commented on index.html:
```diff
@@ -0,0 +1,67 @@
+  <body>
+    <header>
+      <div class="nav-bar">
+        <button type="button" id="applebutton">
+          <i class="fab fa-apple"></i>
+        </button>
+        <button type="button" id="storebutton">Store</button>
+        <button type="button" id="macbutton">Mac</button>
+        <button type="button" id="ipodbutton">iPod</button>
+        <button type="button" id="iphonebutton">iPhone</button>
+        <button type="button" id="ipadbutton">iPad</button>
+        <button type="button" id="itunesbutton">iTunes</button>
+        <button type="button" id="supportbutton">Support</button>
```
- [x] [Required] The first eight elements should be links (to other sections or `#`) ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/))

SpaYco commented on index.html:
```diff
@@ -0,0 +1,67 @@
+        <button type="button" id="ipadbutton">iPad</button>
+        <button type="button" id="itunesbutton">iTunes</button>
+        <button type="button" id="supportbutton">Support</button>
+        <input type="text" id="searchbutton" /><i class="fas fa-search"></i>
```
- [x] [Required] your search icon is flying outside of the input, please fix it ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/))

![image](https://user-images.githubusercontent.com/11758151/90667216-62b77600-e246-11ea-8ee1-d09f3885de78.png)

SpaYco reviewed (changes requested):
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [x] Files not related to the project should not be committed and added to .gitignore (e.g. `.DS_Store`) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)), you have some images that are not used

- I've included more changes down below

## Optional 

- [x]  use the [`<nav>` tag](http://html5doctor.com/nav-element/) for the major navigation links

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

m-anwar97 committed:
deleted unnecessary image

m-anwar97 committed:
changed to nav tag

m-anwar97 committed:
edited the readme file

Nkuria committed:
Corrected pull request issues

m-anwar97 committed:
fixed html error line 18

m-anwar97 committed:
changed the buttons to a tags

m-anwar97 committed:
moved footer a section in css

santiagorodriguezbermudez commented on index.html:
```diff
@@ -0,0 +1,69 @@
+      /></a>
+      <br />
+      <a href="#"><img src="./images/cta_your_verse_27pt.png" alt="#" /></a>
+    </header>
```
- [x] These should not be images but rather h and p tags elements.

santiagorodriguezbermudez commented on style.css:
```diff
@@ -0,0 +1,102 @@
+  background: url("./images/apple2.jpg");
+  background-repeat: no-repeat;
+  min-height: 500px;
+  background-size: 100% auto;
```
- [x] Change background-size to 'cover'. This way the image will always occupy all of its backgrounds.

santiagorodriguezbermudez commented on index.html:
```diff
@@ -0,0 +1,69 @@
+  <body>
+    <header>
+      <nav class="nav-bar">
+        <a href="#"><div id="applebutton"><i class="fab fa-apple"></i></div></a>
+        <a href="#"><div id="storebutton">Store</div></a>
+        <a href="#"><div id="macbutton">Mac</div></a>
+        <a href="#"><div id="ipodbutton">iPod</div></a>
+        <a href="#"><div id="iphonebutton">iPhone</div></a>
+        <a href="#"><div id="ipadbutton">iPad</div></a>
+        <a href="#"><div id="itunesbutton">iTunes</div></a>
+        <a href="#"><div id="supportbutton">Support</div></a>
```
- [x] These ids are not being used. Kindly delete them. Often, prefer use classes over ids.

santiagorodriguezbermudez reviewed (changes requested):
# Changes Required ♻️

Good job implementing the last changes from your last CR. However, there are still a couple of changes you must implement. 

## Required Changes 🤓
- [x] Fixed the padding of your navbar. Moreover, fixed the width of your navbar as well. It should be wider according to the original page. 
![Screen Shot 2020-08-20 at 6 27 18 AM](https://user-images.githubusercontent.com/5817056/90764637-35061780-e2ae-11ea-8cea-a3c23f9ed64b.png)

- [x] The right upper search input element should be embedded inside the element and again a padding and background shadow should be added. 
![Screen Shot 2020-08-20 at 6 28 48 AM](https://user-images.githubusercontent.com/5817056/90764741-6a126a00-e2ae-11ea-8213-498a49c315e3.png)

- [x] The four images on your page have a white space in between them that is currently not in the original page. Kindly, remove this white space. 
<img width="1676" alt="Screen Shot 2020-08-20 at 6 29 55 AM" src="https://user-images.githubusercontent.com/5817056/90764843-a04fe980-e2ae-11ea-94b0-77594505f7f7.png">

- [x] The footer looks messy and missing some bars in between them. As well as the links in their color blue. Again, review the original page. 
![Screen Shot 2020-08-20 at 6 32 34 AM](https://user-images.githubusercontent.com/5817056/90765082-f45ace00-e2ae-11ea-994d-9d89c53dac61.png)

- [x] Your Pull Request must have more information. Every time you provide for new changes, write a descriptive comment of the changes you've implemented since the last CR. This will help you communicate better your changes. 

- [x] Changes over your code (see below)

**Please implement the mentioned changes and submit another code review 🧐**

Happy coding 🙂,
Cheers,
Santiago Rodriguez B.
contact me on slack (@santiagorodriguezb) if you need any help.

m-anwar97 committed:
code review fixes #2 (see description)

Fixed the padding of the navbar.
Added the search element inside the nav bar.
Removed the white spaces between the four images.
Added bars between footer elements, added blue links just like the original page.
Added this pull request comment.
Made the required changes to the code asked by the reviewer

m-anwar97 committed:
fixed linter css error

coldatlas commented:
Fixed the padding of the navbar.
Added the search element inside the nav bar.
Removed the white spaces between the four images.
Added bars between footer elements, added blue links just like the original page.
Added this pull request comment.
Made the required changes to the code asked by the reviewer

kevcap reviewed (changes requested):
# [STATUS] CHANGES REQUIRED♻️

Hi team,  good job on making the previously required changes!
You guys are almost there, but in order to accomplish as the actual page, you are required to do these changes listed below:

## [REQUIRED CHANGES]
- [x] I still can see a white space at the end of those 4 images. That is currently not on the original page. Kindly, remove this white space. 
![image](https://user-images.githubusercontent.com/56762529/90801795-bc727b80-e2ec-11ea-963e-2972b1ed8199.png)

- [x] Please improve the footer 🦶, as requested the previous TSE, to make it more similar to the original one.
* Yours
![image](https://user-images.githubusercontent.com/56762529/90802446-a5805900-e2ed-11ea-9b2a-bf83d2990ede.png)
* Original
![image](https://user-images.githubusercontent.com/56762529/90802472-b16c1b00-e2ed-11ea-852a-87d8a571487f.png)




## [OPTIONAL CHANGES]
 - [ ] Improve your navbar, make it more similar to the original one
* Yours
![image](https://user-images.githubusercontent.com/56762529/90802123-268b2080-e2ed-11ea-8a8b-eb04cf3f7a72.png)
* Original one
![image](https://user-images.githubusercontent.com/56762529/90802162-3276e280-e2ed-11ea-87ad-86bff8adf505.png)
- [ ] Put your `css` files in a css folder. (To make your repo more professional)


## Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> <img  alt="gmail"  src="https://img.shields.io/badge/gmail-%20- green?logo=gmail&style=social"></a> [![Twitter Follow](https://img.shields.io/twitter/follow/kevinnnmn?style=social)](https://twitter.com/intent/follow?screen_name=kevinnnmn) [![](https://img.shields.io/badge/Slack-Kevin%20Alves-green?logo=slack&style=social)](https://microverse-students.slack.com/team/UT86D3AP4)

  
<sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

m-anwar97 committed:
added flag/fixed footer/removed white part

m-anwar97 committed:
fixed offers section

coldatlas commented:
Fixed the white space issue and improved footer

kitpao reviewed (approved):
# 👋 Hello @coldatlas and @Nkuria 

🌟 Great job implementing your last mandatory changes, now it is working as expected 👏 

### 👏 Your project has been approved 👏
### 🚀 You may merge to master and submit the project completition form 🚀

___
##### 👉 If you have any doubts you can contact me in slack as @kitpao

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__
